upf/upf_pfcp_api.c

```diff
@@ -1755,7 +1756,8 @@ handle_create_urr (upf_session_t * sx, pfcp_create_urr_t * create_urr,
 
     create->measurement_period.handle =
       create->time_threshold.handle =
-      create->time_quota.handle = create->traffic_timer.handle = ~0;
+      create->time_quota.handle = create->traffic_timer.handle =
+      create->quota_validity_time.handle = ~0;
```

Comment:
I would reorder this to have quota_validity_time directly after time_quota.

Side Note: it is somewhat strange that the quota_holding_time and monitoring_time is not initialized as well. But that would be a topic for another PR.